src/features/rewards/tipsMigrationAlert/style.ts

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import styled from 'styled-components'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  width: 100%;
+  display: block;
+  background: #E9F0FF;
+  padding: 5px 30px;
+  border-bottom-left-radius: 6px;
```

Comment:
Since this component ends up inside the box, can it just have itself be clipped by its container? I.e. keep the border radius on the Box component, and have this alert be the last child of the Box?

Reply:
@ryanml you could just add border radius to new wrapper that you created for box, so that you can remove this one https://github.com/brave/brave-ui/pull/253/files#diff-6bd2b80c49423c16c202f58fbb1010b3R34

Reply:
done